### ISSUE TITLE
Very slow syncing to iOS

### ISSUE THREAD

**Issue by robe070** [laurent22/joplin#10042]
### Operating system

iOS

### Joplin version

12.13.10

### Current behaviour

The synchronisation on iOS has been running for more than 4 weeks. Sync status report seems to suggest that attachments are still to complete. And the number of notes downloaded grows whilst refreshing Sync Status.

I have encryption on.
And the notebook list is incomplete and the structure of the notebook list is not the same as the desktop. It seems to have not caught up yet.

If I leave Joplin as the foreground app, say on the Sync Status page, progress occurs.

Is it only handling attachments at low priority, or only when I'm using the app, or what?

There is no option in iOS Joplin settings for background operation.
![IMG_1189](https://github.com/laurent22/joplin/assets/4089160/07615901-7d91-4bbe-b1ab-8d6dcf8b5bd9)
![IMG_1190](https://github.com/laurent22/joplin/assets/4089160/6bea9024-7eda-4403-82d4-9409840b339f)

**Comment by laurent22** [comment]
What does the sync status say in the sidebar? Could you provide a screenshot?

With the "always" option it should keep downloading everything in the background, but if a resource is particularly large it might get stuck on it and maybe every time the app is closed it's interrupting that download.

Also are there any errors in the log? https://joplinapp.org/debugging

**Comment by robe070** [comment]
I've tracked down the issue to NOT downloading anything unless Joplin is the foreground app. As soon as the screen is locked, the downloading stops. I was sure that in my initial testing, which may have been more than 6 months ago, that downloading did continue in the background. Definitely not now. (I used Dropbox before. (I'm using Joplin server now) Maybe the Dropbox connection, as its a local file system, stays active in Auto-Lock?)

I've set the Auto-Lock to NEVER and it has been running now constantly for 8 hours and has far surpassed the progress of all the previous 4 weeks combined (I was on holiday and so I didn't look into what was happening)

The decryption also slows everything down a lot.

And not running in the background is far less of an issue if the Cloud connection is external to Joplin. Like if Dropbox was used. Because Dropbox updates in the background.
